ros2_ws/src/OTTO1500_Description/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
```

Comment:
We will probably need to add links for sensors (lidar etc.) later on, but this is ok now.